Disconnect stale client connections after a timeout period

The clients are contacted via the encrypted channel, and the connection is
terminated if the clients fail to respond.

Index: ssh/sshd_config
===================================================================
--- ssh.orig/sshd_config
+++ ssh/sshd_config
@@ -161,4 +161,5 @@ AllowGroups sshlogin
 # macs: anchor_1i2w6k7g8dtsxb62ev3eliju4_18
 # kex: anchor_1i2w6k7g8dtsxb62ev3eliju4_19
 # kick-stale: anchor_1i2w6k7g8dtsxb62ev3eliju4_21
+ClientAliveInterval 1800
 # EOF: anchor_1i2w6k7g8dtsxb62ev3eliju4_20
